### PR TITLE
Add TailWind frontend folder

### PR DESCRIPTION
**Pull request by adamf321**
Added the folder from https://github.com/wearenolte/lean-theme-frontend-tailwind, replacing the existing frontend folder.

Is this correct? Looks like a lot of files have changed @Fcog @nelsonamaya82

### CODE REVIEW

**Comment by Fcog** [comment]
> Added the folder from https://github.com/wearenolte/lean-theme-frontend-tailwind, replacing the existing frontend folder.
> 
> Is this correct? Looks like a lot of files have changed @Fcog @nelsonamaya82

let me download this branch and review

**Comment by Fcog** [comment]
@adamf321 @nelsonamaya82 @bethshook @arzola the integration with tailwind is ready for review.
You can download the branch and test the site.

**Comment by bethshook** [comment]
Yes, I remember there being a lot of changes. In some cases I think some elements are overstyled for a starter theme, like select fields and default blocks like the pull quote.

File: index.php [hunk 1]
```diff
@@ -1,9 +0,0 @@
-<?php
```
Comment (nelsonamaya82) [inline index.php]: @adamf321 @Fcog  why this file was deleted? When I go to Appearance -> Themes I get this error:
`Template is missing. Standalone themes need to have a index.php template file.`

Can I bring it back?

Comment (Fcog) [reply index.php]: yes please @nelsonamaya82 !

**Pull request merged**